qa/backwards/shared/src/test/java/org/elasticsearch/test/ESBackcompatTestCase.java

```diff
@@ -212,17 +196,14 @@ private Settings addLoggerSettings(Settings externalNodesSettings) {
         return finalSettings.build();
     }
 
-    protected int minExternalNodes() { return 1; }
+    protected int minExternalNodes() {
+        return 1;
+    }
 
     protected int maxExternalNodes() {
         return 2;
     }
 
-    @Override
-    protected int maximumNumberOfReplicas() {
```

Comment:
Not sure if this would work, but could you keep this override but make it relative to the number of nodes on latest version, essentially removing the need for hte new "num lucene latest version" method? That is a very awkward parameter for a general integ test that is really specific to backcompat tests...